package.json

```diff
@@ -23,10 +23,8 @@
   },
   "dependencies": {
     "babel-cli": "^6.18.0",
-    "babel-plugin-array-includes": "^2.0.3",
```

Comment:
Hmm, this doesn't matter for this PR, but it would have been better to remove the devDependencies here. The rule I use is - if it's needed to run the site, put it in dependencies, and if it's just needed to debug or test the site, put it in devDependencies. This is a new rule for me - I've only just started using it recently so not all the dependencies currently follow it (which is why we had some duplicates).

In this case, `babel-plugin-array-includes` and `babel-preset-react` are used by babel when compiling the site code when `npm run start` is called, so they aren't just used for debugging/testing.

This is just so that you know in future - please don't change this here - I'll do a cleanup of dependencies in the master branch after the PR is merged.